### DISCUSSION TITLE
Error: `Unable to determine the WEBC file's entrypoint`

### DISCUSSION THREAD

**Discussion by shubham0204** [wasmerio/wasmer#4636] [Q&A] [answered]
I'm using Wasmer 4.2.8 on Windows 11 (x86-64) to try the `sqlite/sqlite` package by executing the command given on the package's registry homepage,

```
$> wasmer run sqlite/sqlite
```

The execution ends with an error,

```
$> wasmer run sqlite/sqlite
error: Unable to determine the WEBC file's entrypoint. Please choose one of ["sqlite", "sqlite3"]
```

Is there anything that should be taken care of, or am I missing something?

**Comment by theduke** [comment, marked as answer]
You can choose the entrypoint with the `-e` flag:

`wasmer run sqlite/sqlite -e sqlite3`

```
  -e, --entrypoint <ENTRYPOINT>
          The function or command to invoke
```
